english/2spooky/ghostbusters.md

```diff
@@ -0,0 +1,8 @@
+
+[The 103 Greatest Quotes From 'Ghostbusters':](http://www.mtv.com/news/1916081/103-ghostbusters-quotes/)
+
+Janine Melnitz: "Do you believe in UFOs, astral projections, mental telepathy, ESP, clairvoyance, spirit photography, telekinetic movement, full trance mediums, the Loch Ness monster, and the theory of Atlantis?"
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.